corehq/apps/users/tests/test_models.py

```diff
@@ -13,6 +14,35 @@
 from corehq.apps.domain.models import Domain
 
 
+class HqPermissionsTest(SimpleTestCase):
```

Comment:
This might fit in better with `test_models_role.py` or `test_user_role.py`, as they have other tests around setting permissions.

Reply:
resolved in [b366970](https://github.com/dimagi/commcare-hq/pull/32345/commits/b36697058214b20564c6519b56405d6d7d302bb5)